### DISCUSSION TITLE
[OKD 4.7 ] Error: unable to load configmap based request-header-client-ca-file

### DISCUSSION THREAD

**Discussion by ingedevops** [okd-project/okd#1559] [General]
**Describe the bug**
The connection to the console interface lasts a little while then I have an error and I can no longer connect, the "api-server" pods are alternately in crashloop back off
Error: unable to load configmap based request-header-client-ca-file 

**Version**
OKD 4.7 on premise

**How reproducible**
Always when trying to connect api console interface

**Log bundle**
$ oc -n openshift-oauth-apiserver get pods
![image](https://user-images.githubusercontent.com/23017591/231146852-80eb63f7-13c2-4ea5-827f-d92860c20d05.png)

 Get "https://IP:443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication?timeout=10s": dial tcp IP:443: connect: connection refused Usage: oauth-apiserver 
 start [flags] Flags: --admission-control-config-file string File with admission control configuration.

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

**Reply by ingedevops** [reply]
unfortunately i don't have it , have some probleme to get it ... .  could you help me anyway with the informations given , i don 't think it is a difficult problem actually i am a beginner in okd , thanks

**Comment by ingedevops** [comment]
na
